common/static/common/js/spec/discussion/view/discussion_inline_view_spec.js

```diff
@@ -24,6 +24,10 @@
                 '  <button class="discussion-show btn btn-brand" data-discussion-id="test-discussion-id">' +
                 '     <span class="button-text">Show Discussion</span>' +
                 '  </button>' +
+                '  <section class="discussion inline-discussion">' +
```

Comment:
I'm not sure why this is necessary. This button is rendered client-side, so you shouldn't need to include it in the server-side mocked response. In fact, it is risky to do this, because the code is not testing with exactly what the server will be rendering.